app/models/non_hmis_assessment.rb

```diff
@@ -110,12 +110,15 @@ def update_assessment_score!
   end
 
   def total_days_homeless_in_the_last_three_years
```

Comment:
@eanders I am not sure if the logic here is correct.

Reply:
Looks right to me!  Thanks.

Reply:
Hmm, it's possible `days_homeless_in_the_last_three_years` should be ignored, that may already be the sum of  `homeless_nights_sheltered` and `homeless_nights_sheltered`.

Reply:
Confirmed, correct as written.  Thanks!